06_puppy/sirigithub/stores_test.go

```diff
@@ -0,0 +1,87 @@
+package main
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/suite"
+)
+
+type storesSuite struct {
+	suite.Suite
+	store     Storer
+	newStorer func() Storer
+	puppy1    Puppy
+}
+
+func (suite *storesSuite) SetupTest() {
+	suite.store = suite.newStorer()
+	suite.puppy1 = Puppy{Breed: "cavalier", Color: "White", Value: "$1700"}
+	suite.store.CreatePuppy(&suite.puppy1)
+}
+func (suite *storesSuite) TestCreatePuppySuccesful() {
+	//given
```

Comment:
nit: I guess some people prefer this given..when..then style of comment, but I don't find it helps. What does help is just a blank line where you have those comments. This creates "paragraphs" in the code, much as we use paragraphs in written text. It breaks it up into related sections. Given that all tests have this sort of structure when we are being consistent then I don't think it needs to be explicitly written out in each test.

Reply:
Agreed. Updated.